package.json

```diff
@@ -11,9 +11,9 @@
     "scripts": {
         "start": "node src/cobalt",
         "setup": "node src/modules/setup",
-        "test": "node src/test/test",
+        "test": "node src/util/test",
         "build": "node src/modules/buildStatic",
-        "testFilenames": "node src/test/testFilenamePresets"
+        "testFilenames": "node src/util/testFilenamePresets"
     },
```

Comment:
why not add a script shortcut for generating a youtube token? :3c

Reply:
idk i dont think its really necessary here

Reply:
testFilenames should probably get removed too

Reply:
it's either both or neither i think

Reply:
i'd remove test filenames but add youtube token generate script tbh